packages/chain-adapters/src/cosmossdk/thorchain/ThorchainChainAdapter.ts

```diff
@@ -14,6 +14,7 @@ import {
 import { toAddressNList } from '../../utils'
 import { bnOrZero } from '../../utils/bignumber'
 import { ChainAdapterArgs, CosmosSdkBaseAdapter } from '../CosmosSdkBaseAdapter'
+import { BuildDepositTxInput } from './types'
 
 // static automatic outbound fee as defined by: https://daemon.thorchain.shapeshift.com/thorchain/constants
```

Comment:
dead link

Reply:
https://github.com/shapeshift/lib/pull/1030/files#r979792342 - unless (until?) this is exposed by `https://daemon.thorchain.shapeshift.com`, going to move this comment over to the `thornode.ninerealms.com` endpoint